include/deal.II/lac/petsc_block_vector.h

```diff
@@ -26,6 +26,7 @@
 #  include <deal.II/lac/block_indices.h>
 #  include <deal.II/lac/block_vector_base.h>
 #  include <deal.II/lac/exceptions.h>
+#include <deal.II/lac/vector_type_traits.h>
```

Comment:
Since the lines above have some spacing `#  include`, could you please do this as well so that its consistent?

Please also adjust the date on this file.